COMPILING.txt

```diff
@@ -117,16 +117,16 @@ irc.freenode.net.
    was to install macports (http://www.macports.org/) and install the
    following ports (sudo port install xxxx):
 
-     assimp
-     pkgconfig
-     freetype
-     libsdl2
-     libsdl2_image
-     libsdl2_sound
-     libsigcxx2
-     libvorbis
-     autoconf
-     automake
+      assimp
+      pkgconfig
+      freetype
+      libsdl2
+      libsdl2_image
+      libsigcxx2
+      libvorbis
+      autoconf
+      automake
+
 
```

Comment:
Nitpicking: Looks like your editor has inserted tabs (or changed indentation for some reason). Could you please untabify it?